source/docs/installation/installation-nonfrc.rst

```diff
@@ -10,24 +10,155 @@ Phoenix 6 is fully supported outside of `FRC <https://en.wikipedia.org/wiki/FIRS
+
+.. note:: ``<year>`` should be replaced with the year of Phoenix 6 software for which you have purchased licenses.
+
+Open ``/etc/apt/sources.list.d/ctr${YEAR}.list`` with an editor of your choice.
+
+.. code-block:: bash
+
+   sudo nano /etc/apt/sources.list.d/ctr${YEAR}.list
+
+Replace ``stable`` with the distribution you want to use.
+
+.. code-block:: bash
+
+   # APT repo for CTR tools, including canivore-usb
+   deb [signed-by=/usr/share/keyrings/ctr-pubkey.gpg] https://deb.ctr-electronics.com/tools stable main
+
+For example. Raspberry Pi would look like:
+
+.. code-block:: bash
+
+   # APT repo for CTR tools, including canivore-usb
+   deb [signed-by=/usr/share/keyrings/ctr-pubkey.gpg] https://deb.ctr-electronics.com/tools raspberrypi main
+
+Available distributions can be found in the dropdown below.
+
+.. _canivore-modifying-list-dist:
+
+.. dropdown:: Modifying ``.list`` Distribution
+
+   .. list-table::
+      :header-rows: 1
+      :align: left
+      :width: 100%
+
+      * - System
+
+        - ``<dist>``
+
+      * - Raspberry Pi
+
+        - raspberrypi
+
+      * - NVIDIA Jetson
+
+        - jetson
+
+      * - Other Linux systems (default)
+
+        - stable
+
+   .. warning:: Do not modify the distribution of the ``libs/<year>`` entry in the ``.list`` file.
+
+
+CANivore Installation
+^^^^^^^^^^^^^^^^^^^^^
+
+On non-FRC Linux systems, the ``canivore-usb`` kernel module must be installed to add SocketCAN support for the CANivore. The kernel module is distributed through our APT repository.
+
+.. note:: **Custom bit rates and CAN 2.0 are not supported at this time.** The parameters passed into SocketCAN are not applied by the firmware.
 
-         YEAR=<year>
-         sudo curl -s --compressed -o /usr/share/keyrings/ctr-pubkey.gpg "https://deb.ctr-electronics.com/ctr-pubkey.gpg"
-         sudo curl -s --compressed -o /etc/apt/sources.list.d/ctr${YEAR}.list "https://deb.ctr-electronics.com/ctr${YEAR}.list"
+.. dropdown:: Raspberry Pi
 
-      .. note:: ``<year>`` should be replaced with the year of Phoenix 6 software for which you have purchased licenses.
+   .. warning:: Raspberry Pi 4 with 32-bit OS require additional instructions. See :ref:`docs/installation/installation-nonfrc:raspberry pi errata` for more information.
+
+   1. Install the Raspberry Pi kernel headers.
+
+   .. code-block:: bash
+
+      sudo apt install raspberrypi-kernel-headers
+
+   2. Update APT and install ``canivore-usb``
+
+   .. code-block::
+
+      sudo apt update
+      sudo apt install canivore-usb -y
+
+.. dropdown:: Other Supported Distributions
+
+   1. Update APT and install ``canivore-usb``.
+
+   .. code-block:: bash
+
+      sudo apt update
+      sudo apt install canivore-usb
+
+.. tip:: To get a robot application up and running quickly, check out our `non-FRC Linux example <https://github.com/CrossTheRoadElec/Phoenix6-Linux-Example>`__.
+
+Troubleshooting
+^^^^^^^^^^^^^^^
```

Comment:
This section seems kind of out-of-place right now, as it's sitting right in the middle of the installation section (between installing CANivore and installing API).